pkg/volume/fc/fc_util.go

```diff
@@ -178,8 +187,18 @@ func (util *FCUtil) AttachDisk(b fcDiskMounter) (string, error) {
 }
 
 func (util *FCUtil) DetachDisk(c fcDiskUnmounter, mntPath string) error {
+	devName, _, err := mount.GetDeviceNameFromMount(c.mounter, mntPath)
```

Comment:
could there be a race window where the disk to be detached happens to be being attached?